data/enterprise-nav-data.json

```diff
@@ -1,6 +1,7 @@
 [
   { "heading": "Deployment & Operation" },
   { "title": "Overview", "path": "" },
+  { "title": "Operational Modes", "path": "operational-modes" },
```

Comment:
Thinking that maybe it should be (with the new sidebar): 
- Overview
- Requirements
- Operational Mode
- Reference Architectures

Thoughts?